subcommand/connect-init/command_test.go

```diff
@@ -40,18 +61,249 @@ func TestRun_FlagValidation(t *testing.T) {
 	}
 }
 
-// TestRun_RetryACLLoginFails tests that after retries the command fails
+// This test mocks ACL login and the service list response (/v1/agent/services),
+// the later of which is also validated by an actual agent in TestRun_happyPathNoACLs().
+func TestRun_happyPathACLs(t *testing.T) {
+	bearerFile := common.WriteTempFile(t, "bearerTokenFile")
```

Comment:
```suggestion
      t.Parallel()
	bearerFile := common.WriteTempFile(t, "bearerTokenFile")
```

Reply:
with `t.Parallel()` If I run 2 instances of the mock consulServer at the same time and both are watching for the same URL and Method will they not collide?

Reply:
OK I did a bit more investigation and the `httptest` server does pick clean ports so we're good to use `t.Parallel()` thanks for the suggestion!